src/TimeGridHeader.js

```diff
@@ -128,21 +119,58 @@ class TimeGridHeader extends React.Component {
     )
   }
 
+  renderHeaderByDay() {
+    let { resources, range, getNow, accessors } = this.props
+    const today = getNow()
+
+    return range.map((date, index) => (
+      <div className="rbc-time-header-content" key={index}>
+        <div className="rbc-row rbc-time-header-cell">
+          {this.renderHeaderCell({ date, index, today })}
+        </div>
+        <div className="rbc-row rbc-row-resource">
+          {resources.map(([id, resource], idx) => (
+            <div key={`resource_${id || idx}`} className="rbc-header">
+              {accessors.resourceTitle(resource)}
+            </div>
+          ))}
+        </div>
+        {/* TODO - render date content row */}
```

Comment:
I need some advice on how to refactor existing code to allow render DateContentRow to support day/resource grouping